md2conf/application.py

```diff
@@ -15,16 +23,70 @@ def update_document(
     api.update_page(document.page_id, document.xhtml())
 
 
+
+def synchronize_pages(api: ConfluenceSession, path: str, options: ConfluenceDocumentOptions) -> None:
```

Comment:
We might want to wrap this (and other related functions) in a new class so that we pass the parameters once (in the new class `__init__`) and then access them via `self` in member functions like `synchronize_pages`, `synchronize_page`, etc., and don't need to propagate them below.